docs/install/index.md

```diff
@@ -1,25 +1,122 @@
 ---
 title: Install ORT
-parent: How to
+has_children: false
 nav_order: 1
 ---
 
-# Install ONNX Runtime
+# Install ONNX Runtime (ORT)
 {: .no_toc }
 
 See the [installation matrix](https://onnxruntime.ai) for recommended instructions for desired combinations of target operating system, hardware, accelerator, and language. 
 
-Details on OS versions, compilers, language versions, dependent libraries, etc can be found under [Compatibility](../resources/compatibility.md#Environment-compatibility).
+Details on OS versions, compilers, language versions, dependent libraries, etc can be found under [Compatibility](../reference/compatibility).
+
 
 ## Contents
 {: .no_toc }
 
 * TOC placeholder
 {:toc}
 
-## Inference
+## Python Installs
+
+### Install ONNX Runtime (ORT)
+{: .no_toc }
+
+```bash
+pip install onnxruntime
+```
+```bash
+pip install onnxruntime-gpu
+```
+
+### Install ONNX to export the model
+{: .no_toc }
+
+```bash
+## pytorch
+pip install onnx-pytorch
```

Comment:
not sure what this is.. torch.onnx.export is built into PyTorch. no need to install anything